src/context/viewer-context.tsx

```diff
@@ -79,7 +82,23 @@ function useAuthedViewer() {
       }
     }
 
-    if (viewerIsPresent) {
+    if (authToken) {
```

Comment:
This new case at the front of the if/then/else checks if there is an authToken (from a cookie) and then attempts to set up the session with it.